### DISCUSSION TITLE
Timeout error loading app homepage with API http://budibase.lp.corp/api/self

### DISCUSSION THREAD

**Discussion by luigiPinnaAevo** [Budibase/budibase#11978] [Help]
Hello,

I'm currently encountering a timeout problem when trying to load the homepage of my application in Budibase. Has anyone else experienced this issue, and if so, do you have any suggestions for a resolution?

![image](https://github.com/Budibase/budibase/assets/99037506/e73cfa47-c36f-4c2b-8214-ff7a4e88c468)

Thank you

**Comment by ConorWebb96** [comment]
Hey @luigiPinnaAevo,

Can you try exporting and re-importing your app. It looks like there may be an issue with your self api call. Which i believe just pulls in the app current logged in user.

- Does this occur in the builder or your published app? 
- Do you have any errors with your users table whenever checking it in the data section?
- When did you first start noticing this issue?
- What version of Budibase were you on when everything was loading?
- Was there any changes you made to the users table around this time which could have caused the issue?

Any information you feel would be helpful, could you please provide. 

Thanks in advance!

**Reply by luigiPinnaAevo** [reply]
Ok @ConorWebb96, below the answers:

- Does this occur in the builder or your published app? 
In both cases

- Do you have any errors with your users table whenever checking it in the data section? 
No, there are no errors in the user table.

- When did you first start noticing this issue?
I noticed a gradual slowdown.

- What version of Budibase were you on when everything was loading?
It doesn't depend on the versions because I've changed several and the issue didn't occur after any update.

- Was there any changes you made to the users table around this time which could have caused the issue?
No, no changes were made.

Thank you

**Reply by luigiPinnaAevo** [reply]
Exporting and re-importing the app works but I lost the documents saved

**Comment by andyburgessmd** [comment]
Could be similar to my bug
https://github.com/Budibase/budibase/issues/11855 and the discussion https://github.com/Budibase/budibase/discussions/11832
